modules/i3s/test/lib/utils/load-utils.js

```diff
@@ -0,0 +1,57 @@
+import {I3SLoader} from '@loaders.gl/i3s';
+import {Tileset3D, Tile3D} from '@loaders.gl/tiles';
+import I3SNodePagesTiles from '../../../src/helpers/i3s-nodepages-tiles';
+
+const TILESET_STUB = {
```

Comment:
Comment explaining what this is - a loaded tileset? This can't be a file in test data?

Reply:
This can't be a file because in that case we need:
`/path/to/dataset/SceneServer/layers/0` and
`/path/to/dataset/SceneServer/layers/0/nodepages/0` 
(`/path/to/dataset/SceneServer/layers/0` - file and folder)
It is not possible in file system - only in http service.

Reply:
I see, directory and file with the same name? 

I suppose we could modify the file naming slightly when reading from disk. call it `0.json` or some other extension.